movai_core_shared/core/zmq_client.py

```diff
@@ -60,7 +61,9 @@ def send(self, msg: dict) -> None:
             with self.lock:
                 self._socket.send(data)
         except (json.JSONDecodeError, TypeError) as error:
-            self._logger.error(f"Got an {error} while trying to send message")
+            self._logger.error(
+                f"Got error of type {error.__class__.__name__} while trying to send message"
```

Comment:
why not use `type(error)`?

Reply:
becuase the output of type(<classname>) is => <class 'classname'>